system-tests/test/navigation_spec.ts

```diff
@@ -0,0 +1,51 @@
+import systemTests, { expect } from '../lib/system-tests'
+
+const PORT = 13370
+const onServer = function (app) {
+  app.get('/cross_origin.html', (req, res) => {
+    res.send('<html><h1>cross origin</h1></html>')
+  })
+}
+
+// FIXME: This partially solves https://github.com/cypress-io/cypress/issues/19632 but only when "experimentalMultiDomain" is false.
+// TODO: This will be further solved by https://github.com/cypress-io/cypress/issues/20428
+describe('e2e cross origin navigation', () => {
+  systemTests.setup({
+    servers: [{
+      port: 4466,
+      onServer,
+    }],
+    settings: {
+      hosts: {
+        '*.foobar.com': '127.0.0.1',
+      },
+    },
+  })
+
+  // FIXME: add test that supports this for firefox
+  systemTests.it('captures cross origin failures when "experimentalMultiDomain" config value is falsy', {
+    // keep the port the same to prevent issues with the snapshot
+    port: PORT,
+    spec: 'navigation_cross_origin_errors.ts',
+    browser: ['chrome', 'electron'],
+    snapshot: true,
+    expectedExitCode: 1,
+    config: {
+      experimentalMultiDomain: false,
+    },
+    async onRun (exec) {
+      const res = await exec()
+
+      expect(res.stdout).to.contain('Cypress detected a cross origin error happened on page load')
```

Comment:
Are these expects needed if the snapshot is being validated?

Reply:
They aren't needed, but it suggests what is important in the test/snapshot. That way, if a mismatch happens, it hopefully points the dev in the right direction.